src/main/java/exception/AnonymousException.java

```diff
@@ -0,0 +1,27 @@
+package exception;
+
+/**
+ * Thrown to notify user that enter a
```

Comment:
Will it be better to start the class description with the class name here? I have also noticed similar problems in several other places.